CHANGELOG.rst

```diff
@@ -16,6 +16,8 @@ New hooks
   <https://github.com/pyinstaller/pyinstaller-hooks-contrib/issues/83>`_)
 * Added a hook for the 'pygraphviz' library (`#86
   <https://github.com/pyinstaller/pyinstaller-hooks-contrib/issues/86>`_)
+* Add a hook for ``Office365-REST-Python-Client`` which uses data files in
+  some methods
```

Comment:
This isn't how our changelog works. Editing the changelog directly invariably leads to merge conflicts when multiple PRs add to the same file in the same place. Instead write a [news entry](https://github.com/pyinstaller/pyinstaller-hooks-contrib/blob/master/news/README.txt).

Reply:
Apologies for that, I had gotten confused by the instructions in the developer guide. Do you think we could rework that a bit to clarify expectations? I'd be happy to take a stab at revising it, just not sure how to do so.

Reply:
Hmm, I've got a half written branch collecting dust somewhere which would do it if I ever finish it. It's also going to explain how to write tests (which I see you've had to guess at - although you guessed right), how to trigger our custom CI/CD, and the copyright header (which most people forget).